composer.json

```diff
@@ -13,9 +13,10 @@
     },
     "require": {
       "php": "7.*",
-      "automattic/jetpack-connection": "1.30.13",
+      "automattic/jetpack-connection": "1.33.0",
```

Comment:
Needs this update, otherwise, `composer update` will yell this incompatible package version error: 

>  automattic/jetpack-sync v1.27.6 requires automattic/jetpack-connection ^1.33 -> found automattic/jetpack-connection[dev-master, v1.33.0, 1.33.x-dev (alias of dev-master)] but it conflicts with your root composer.json require (1.30.13).